charts/openshift-metering/templates/presto/_presto_helpers.tpl

```diff
@@ -45,6 +45,37 @@ connector.name=blackhole
 connector.name=memory
 {{ end }}
 
+{{- define "presto-prometheus-catalog-properties" -}}
+{{- if .Values.presto.spec.config.connectors.prometheus.enabled }}
+{{- with .Values.presto.spec.config.connectors.prometheus -}}
+connector.name=prometheus
+{{- if .config.uri }}
+prometheus-uri={{ .config.uri }}
+{{- else }}
+prometheus-uri=http://localhost:9090
+{{- end }}
+{{- if .config.chunkSizeDuration }}
+query-chunk-size-duration={{ .config.chunkSizeDuration }}
+{{- else }}
+query-chunk-size-duration=1h
+{{- end }}
+{{- if .config.maxQueryRangeDuration }}
+max-query-range-duration={{ .config.maxQueryRangeDuration }}
+{{- else }}
+max-query-range-duration=1d
+{{- end }}
+{{- if .config.cacheDuration }}
+cache-duration={{ .config.cacheDuration }}
+{{- else }}
+cache-duration=30s
+{{- end }}
+{{- if or .auth.useServiceAccountToken .auth.bearerTokenFile }}
+bearer-token-file={{ .auth.bearerTokenFile }}
+{{- end }} {{- /* end-if */ -}}
+{{- end }} {{- /* end-with */ -}}
+{{- end }} {{- /* end-if-enabled */ -}}
+{{- end }} {{- /* end-define */ -}}
```

Comment:
When viewing the `prometheus.properties` in the presto-coordinator container, there's no trailing newline like the other connector properties files. Replacing `{{- end }}` with `{{ end }}` from lines 74 to 77 will fix this problem.

Reply:
Done in d98d9846.